README.md

```diff
@@ -12,100 +12,80 @@
 
 # Data Load Tool (DLT)
 
-Data Load Tool (DLT) enables simple, python-native data pipelining for data professionals.
+## What is DLT?
 
-It is an open source, scalable data loading framework that does not require any help from DevOps.
+Data Load Tool (DLT) is an open source python library for building data pipelines.
 
-Learn more with the **[Quickstart Guide](QUICKSTART.md)** and check out **[Example Sources](examples/README.md)** to get started.
-
-## How does it work?
-
-DLT aims to simplify data loading for everyone.
+It's goal is to enable faster building of low-maintenance data pipelines.
 
+It's designed to run anywhere (python 3.8+) and it can fit in your existing workflows or be scheduled independently.
```

Comment:
```suggestion
It's designed to run anywhere (Python 3.8+), so it can fit into your existing workflows or be scheduled independently (e.g. using GitHub Actions)
```